eva-lib/src/main/java/uk/ac/ebi/eva/lib/filter/VariantEntityRepositoryFilter.java

```diff
@@ -27,13 +27,13 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-public class VariantRepositoryFilter<T> {
+public class VariantEntityRepositoryFilter<T> {
```

Comment:
I think it narrows too much it's usefulness and, if we want  to limit which filters can be used in a repository it would be better to use interfaces to mark which ones can be used, that way we can reuse a filter in multiple repositories.